### ISSUE TITLE
Filtering by Dose Form

### ISSUE THREAD

**Issue by ericaVoss** [OHDSI/Atlas#291]
When searching the Vocabulary and/or making a concept set, I'd like to be able to filter to drugs of a certain dose form.  For example, I need oral corticosteroids.  I know how to query the Vocabulary for that but I cannot do that from in the tool currently.

Knowing that all dose forms are not available it would be a dream to be able to exclude/include descriptions containing text like "oral".

```{SQL}
WITH CTE_DRUGS AS (
	SELECT DISTINCT c2.*
	FROM CONCEPT c
		JOIN CONCEPT_ANCESTOR cr
			ON cr.ANCESTOR_CONCEPT_ID = c.CONCEPT_ID
		JOIN CONCEPT c2
			ON c2.CONCEPT_ID = cr.DESCENDANT_CONCEPT_ID
	WHERE c.CONCEPT_ID IN (
		4333495,21602722,723814,21502569,4284007
	)
)
SELECT d.CONCEPT_ID, d.CONCEPT_NAME, c.CONCEPT_ID, c.CONCEPT_NAME
FROM CTE_DRUGS d
	LEFT OUTER JOIN CONCEPT_RELATIONSHIP cr
		ON cr.CONCEPT_ID_1 = d.CONCEPT_ID
		AND cr.RELATIONSHIP_ID = 'RxNorm has dose form' 
	LEFT OUTER JOIN CONCEPT c
		ON c.CONCEPT_ID = cr.CONCEPT_ID_2
WHERE d.CONCEPT_NAME LIKE '%oral%'
OR c.CONCEPT_ID IN (
		1841890,1842126,1842942,1842949,1843016,1843853,1843857,1843870,1843879,1843998,19001949,19082076,19082077,19082079,19082168,19082170,19082191,19082253,19082255,19082573,19095918,44535484,44535487,44535504,44535526,44535533,44535536,44535543
)
ORDER BY 1
```

**Comment by cgreich** [comment]
We do have a mapping from Dose Form to Route of Administration somewhere. I'll dig it up. It isn't completely unambiguous, but good enough for most purposes.

**Comment by vojtechhuser** [comment]
I would say +1 to the problem of  topical vs systemic. (in the long run it would a nice to have feature in the ecosystem)

**Comment by chrisknoll** [comment]
@ericaVoss , I looked at the tool:  does searching for the ingredient in question and then looking at the class 'Clinical Drug Form' work?  I did a test using Methotrexate, and then filtered by clinical drug form, and I found Oral solution, oral tablet, oral suspension (along with other forms). What I've done in the past is selected the forms of interest for my concept set and then added descendants.
